frontend/src/app/leaderboard/Nav.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
```

Comment:
adapt this to the already exisiting nav used in /app/profile 

do not create a new one

Reply:
I'm not entirely sure what is meant by "adapt this to the already existing nav used in /app/profile." The Figma design for the leaderboard navbar is different from the one in /app/profile.